### PR TITLE
Adding Synth and Keys' Debit Card

### PR DESCRIPTION
**Pull request by Synth-and-Keys**
Initial Commit

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/synthandkeys/the_debit_card/config.h [hunk 1]
```diff
@@ -0,0 +1,20 @@
+// Copyright 2022 Synth and Keys (@Synth-and-Keys)
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/config.h:1]: This file should be removed

File: keyboards/synthandkeys/the_debit_card/info.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    },
+    "processor": "RP2040",
+    "url": "",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0000",
+        "vid": "0xFEED"
+    },
+    "layouts": {
+        "LAYOUT_ortho_4x4": {
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/info.json:30]:
```suggestion
        "LAYOUT: {

```

File: keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,19 @@
+#include QMK_KEYBOARD_H
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c:1]:
```suggestion
// Copyright 2022 Synth and Keys (@Synth-and-Keys)
// SPDX-License-Identifier: GPL-2.0-or-later
#include QMK_KEYBOARD_H
```

File: keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ 0 │
+     * ├───┼───┼───┼───┤
+
+     */
+    [0] = LAYOUT_ortho_4x4(
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c:14]:
```suggestion
    [0] = LAYOUT(
```

File: keyboards/synthandkeys/the_debit_card/readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# synthandkeys/the_debit_card
+
+![synthandkeys/the_debit_card](https://imgur.com/a/HupOLSZ)
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/readme.md:3]:
```suggestion
![synthandkeys/the_debit_card](https://i.imgur.com/8dpEv73.jpeg)

```

File: keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    /*
+     * ┌───┬───┬───┬───┐
+     * │ 7 │ 8 │ 9 │   │
+     * ├───┼───┼───┼───┤
+     * │ 4 │ 5 │ 6 │   │
+     * ├───┼───┼───┼───┤
+     * │ 1 │ 2 │ 3 │ 0 │
+     * ├───┼───┼───┼───┤
+
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/keymaps/default/keymap.c:12]:
```suggestion
     * ┌───┬───┬───┐
     * │ 7 │ 8 │ 9 │
     * ├───┼───┼───┤
     * │ 4 │ 5 │ 6 │
     * ├───┼───┼───┼───┐
     * │ 1 │ 2 │ 3 │ 0 │
     * └───┴───┴───┴───┘
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by Synth-and-Keys** [comment]
Completed requested changes concerning files

File: keyboards/synthandkeys/the_debit_card/info.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
+{
+    "manufacturer": "Synth and Keys",
+    "keyboard_name": "synthandkeys/the_debit_card",
+    "maintainer": "Synth-and-Keys",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
```
Comment (zvecr) [inline keyboards/synthandkeys/the_debit_card/info.json:6]:
```suggestion
```

**Review by zvecr — APPROVED** [review]

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**